expression/builtin_string_vec.go

```diff
@@ -0,0 +1,65 @@
+package expression
+
+import (
+	"math"
+	"strings"
+
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+func (b *builtinRepeatSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf, err := b.get(types.ETString, n)
```

Comment:
`b.get` is ambiguous to me. Could it be `b.bufAllocator.get` and disallow someone to call it directly?

Reply:
`get/put` is OK for me... what's your opinions? @zz-jason @XuHuaiyu

Reply:
updated, PTAL @SunRunAway